compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -48,9 +49,22 @@ import DA.Daml.Preprocessor
 import Development.IDE.GHC.Util
 import qualified Development.IDE.Types.Options as Ghcide
 
+mkPackageFlag :: UnitId -> PackageFlag
+mkPackageFlag unitId = ExposePackage "--package " (UnitIdArg unitId) (ModRenaming True [])
+
+mkBaseUnits :: Maybe String -> [UnitId]
+mkBaseUnits optMbPackageName
+  | optMbPackageName == Just "daml-prim" =
+      []
+  | optMbPackageName == Just damlStdlib =
+      [ stringToUnitId "daml-prim" ]
+  | otherwise =
+      [ stringToUnitId "daml-prim"
+      , stringToUnitId damlStdlib ]
+
 -- | Convert to ghcide’s IdeOptions type.
 toCompileOpts :: Options -> Ghcide.IdeReportProgress -> Ghcide.IdeOptions
-toCompileOpts options@Options{..} reportProgress =
+toCompileOpts options@Options{..} reportProgress = do
```

Comment:
```suggestion
toCompileOpts options@Options{..} reportProgress =
```
no need to make the diff unnecessarily noisy and this `do` doesn’t make any sense.

Reply:
done
(there was `let` added here, but it got moved in `getDamlGhcSession` after the last rebase)